doc/man3/EVP_EncryptInit.pod

```diff
@@ -274,6 +275,11 @@ is not supported.
 EVP_CIPHER_CTX_ctrl() allows various cipher specific parameters to be determined
 and set.
 
+EVP_CIPHER_CTX_rand_key() generates a random key of size
```

Comment:
I literraly meant 
```
B<Key> must point to a buffer at least as big as the value returned by
EVP_CIPHER_CTX_key_length().
```

Reply:
ok , misunderstood. added this sentence to the original submission.